### PR TITLE
Feat revamp game

### PR DESCRIPTION
**Pull request by tyrossel**

### CODE REVIEW

**Review by YvanoffP — APPROVED** [review]

File: frontend/src/components/alert/AlertBox.vue [hunk 1]
```diff
@@ -82,6 +82,16 @@ const handleSubmit = (e?: Event) => {
         })
     }
 }
+
+const handleRefuse = (e?: Event) => {
+	console.log(`handle refuse`)
```
Comment (Bima42) [inline frontend/src/components/alert/AlertBox.vue:87]: console.log

File: frontend/src/game/scenes/UiScene.ts [hunk 2]
```diff
@@ -56,19 +27,19 @@ export default class UiScene extends Phaser.Scene {
 
 	}
 
-	create(config: IGameSettings) {
+	create() {
 		// this.modal = new Modal("");
-		this.gameSettings = config;
-		this.scoreWidget = this.add.text(0, 50, "0 - 0", { fontFamily: 'Arial', fontSize: "25px", color: "#00FF00" });
+		this.gameSettings = gameStore.currentGame
+		this.scoreWidget = this.add.text(0, 50, "Press W or S to move your paddle", { fontFamily: 'Arial', fontSize: `${pong.worldWidth * 0.03}px`, color: "#00FF00" });
```
Comment (Bima42) [inline frontend/src/game/scenes/UiScene.ts:33]: martial

File: frontend/src/views/JoinAGameView.vue [hunk 3]
```diff
@@ -72,20 +72,19 @@ function joinQueue() {
...
 
-	gameStore.socket.once('matchFound', () => {
+	gameStore.socket.once('matchFound', (gameSettings: IGameSettings) => {
 		isLoading.value = false;
 		gameStore.socket.off("invitationDeclined")
+		gameStore.currentGame = gameSettings
 		router.push(`game`);
 	});
 	gameStore.socket.once("invitationDeclined", (gameSettings: IGameSettings) => {
+		console.log(`invitation declined :(`)
```
Comment (Bima42) [inline frontend/src/views/JoinAGameView.vue:82]: console

**Review by Bima42 — CHANGES_REQUESTED** [review]

File: frontend/src/game/scenes/UiScene.ts (hunk 2 above)
Comment (tyrossel) [reply frontend/src/game/scenes/UiScene.ts:33]: wont change for now

**Pull request merged**